bears/js/JSStandardBear.py

```diff
@@ -16,13 +17,36 @@ class JSStandardBear:
     """
 
     LANGUAGES = {'JavaScript', 'JSX'}
-    REQUIREMENTS = {NpmRequirement('standard', '10')}
+    REQUIREMENTS = {NpmRequirement('standard', '10.0.3'),
+                    NpmRequirement('babel-eslint', '8'),
+                    NpmRequirement('eslint-plugin-flowtype', '2.41.0'),
+                    NpmRequirement('typescript-eslint-parser', '12.0.0'),
+                    NpmRequirement('eslint-plugin-typescript', '0.8.1')}
```

Comment:
those are `eslint` requirements, they don't affect `standard`, do they?

Reply:
we do require them for checking typescript and flow with standard

Reply:
yes... https://standardjs.com/index.html#flow